subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GradleNodePluginSmokeTest.groovy

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.smoketests
+
+class GradleNodePluginSmokeTest extends AbstractPluginValidatingSmokeTest {
```

Comment:
Can we/should we merge the test with `NodePluginsSmokeTest`? I think it would be better to see all the node version in one place.

Reply:
done